cmd/build.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+DIR=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd );
+
+PLACEHOLDER_DATA=${PLACEHOLDER_DATA:-"./data"};
```

Comment:
It's a default value to be used if the variable on the left side of the brackets is empty: http://www.tldp.org/LDP/abs/html/parameter-substitution.html